scripts/component_readme/readme_template.md

```diff
@@ -1,31 +1,45 @@
 # {{ name }}
 
-### Description
+## Description
 {{ description }}
 
-### Inputs / outputs
+## Inputs / outputs
 
+### Consumes
 {% if consumes %}
 **This component consumes:**
 
 {% for field_name, field in consumes.items() %}
 - {{ field.name }}: {{ field.type.value }}
 {% endfor %}
 {% else %}
-**This component consumes no data.**
+_**This component does not consume specific data.**_
 {% endif %}
 
+{% if is_consumes_generic %}
+**This component consumes generic data**
+- <field_name>: <mapped_field_name>
+{% endif %}
```

Comment:
I don't think "does not consume specific data", or "consumes generic data" is clear for users without deep knowledge about Fondant concepts. I think we can refactor this and update the messages to be more clear:

```
{% if consumes %}
**This component consumes:**

{% for field_name, field in consumes.items() %}
- {{ field.name }}: {{ field.type.value }}
{% endfor %}
{% elif is_consumes_generic %}
**This component can consume different data schemes**
See the usage example below on how to define a schema.
{% else %}
**This component does not consume data.**
{% endif %}
```